src/conf_mode/interfaces-openvpn.py

```diff
@@ -274,7 +275,8 @@ def verify(openvpn):
             elif v6remAddr and not v6loAddr:
                 raise ConfigError('IPv6 "remote-address" requires IPv6 "local-address"')
 
-            if (v4loAddr == v4remAddr) or (v6remAddr == v4remAddr):
+            if ((is_list_equal(v4loAddr, v4remAddr) and v4loAddr == v4remAddr) or
```

Comment:
This now feels a bit redundant as you already to the `sorted(first) == sorted(second)` check in the helper.

`if is_list_equal(v4loAddr, v4remAddr) or is_list_equal(v6loAddr, v6remAddr):` should be sufficient.

Reply:
Done!